airflow/models.py

```diff
@@ -1650,22 +1650,6 @@ def signal_handler(signum, frame):
                 if result is not None:
                     self.xcom_push(key=XCOM_RETURN_KEY, value=result)
 
-                # TODO remove deprecated behavior in Airflow 2.0
-                try:
-                    task_copy.post_execute(context=context, result=result)
```

Comment:
I think the deprecated bit is post_execute with one arg - we still want this call with two.

Reply:
Whoops, too fast, thanks @ashb